emergent/memory.py

```diff
@@ -3,10 +3,9 @@
 from collections import defaultdict
 from datetime import datetime
 import json
-
+import logging
 from openai.embeddings_utils import cosine_similarity
-
-from .llms import chat_gpt_prompt, get_embedding
+from llms import chat_gpt_prompt, get_embedding
```

Comment:
btw, you can install the package locally with the editable flag, by doing `pip install -e .` that way, even if you edit the code, when you want to test it out, you can still import stuff like `from emergent import ChatAgent` and everything will work, like how I wrote some stuff in the tests folder.